mpact/services.py

```diff
@@ -22,39 +21,40 @@
     STATUS,
     TELEGRAM_API_HASH,
     TELEGRAM_API_ID,
+    TOKEN,
     TWO_FA_ENABLED,
     USERNAME,
 )
 from django.contrib.auth.models import User
 from logger import logger
 from rest_framework import status
+from telegram_bot.settings import container
 from telethon import TelegramClient
 from telethon.errors import (
     PhoneCodeExpiredError,
     PhoneCodeInvalidError,
     SessionPasswordNeededError,
 )
-from utils import get_or_none
+from utils import encode_token, get_or_none
 
 from .models import ChatBot, Message
 from .serializers import ChatBotSerializer, MessageSerializer
 
 
 @asynccontextmanager
-async def get_anon_client():
+async def get_anon_client(phone):
```

Comment:
You might want to change this function's name, because it no longer returns a client for "anon". Maybe "get_connected_client" or "client_context"?